ts/card-info/CardStats.svelte

```diff
@@ -105,7 +105,14 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 
         statsRows.push({ label: tr2.cardStatsCardTemplate(), value: stats.cardType });
         statsRows.push({ label: tr2.cardStatsNoteType(), value: stats.notetype });
-        statsRows.push({ label: tr2.cardStatsDeckName(), value: stats.deck });
+        let deck: string;
+        if (stats.deck !== stats.originalDeck) {
+            deck = `${stats.deck} (${stats.originalDeck})`;
+        } else {
+            deck = stats.deck;
+        }
```

Comment:
Making originalDeck optional would be slightly more elegant than comparing the two deck names, but it's not particularly important - let me know if you want this merged without that change.